playwright/tests/rex-test/Base.page.ts

```diff
@@ -0,0 +1,15 @@
+// Base page locators and functions
+
+class BasePage{
+    page: any;
+    constructor(page){
+        this.page = page;
+    }
+
+    // Open a Rex page with base url
+    async navigate(){
+        await this.page.goto('/books/business-ethics/pages/preface')
```

Comment:
instead of 'navigate,' rename it as 'open' and allow it to take a string argument; pass the argument to `goto` replacing the Business Ethics URL fragment